### PR TITLE
touchegg: update to 2.0.9

### PR DESCRIPTION
**Pull request by crater2150**
Touchegg was completely rewritten without Qt for 2.0, so most dependencies changed. It also changed to a client-server architecture for access to input devices, so this package now adds a `_touchegg` user and a service running with `input` group.

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me (although I did not use the previous version, as it segfaulted for me)
- [ ] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 
(Please choose at least one native build and, if supported, at least one cross build. More are better.)
- [x] I built this PR locally for my native architecture, (x86_64)
- [x] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [x] aarch64-musl (crossbuild)
  - [x] armv7l (crossbuild)
  - [x] armv6l-musl  (crossbuild)

### CODE REVIEW

File: srcpkgs/touchegg/template [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-build_wrksrc=touchegg
-build_style=qmake
-hostmakedepends="qt5-qmake qt5-host-tools"
-makedepends="libXtst-devel geis-devel qt5-devel qt5-x11extras-devel"
+version=2.0.9
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="libudev-devel libinput-devel pugixml-devel cairo-devel libX11-devel
+	libXtst-devel libXrandr-devel libXi-devel glib-devel gtk+3-devel"
```
Comment (ericonr) [inline srcpkgs/touchegg/template:8]:
```suggestion
 libXtst-devel libXrandr-devel libXi-devel glib-devel gtk+3-devel"
```

File: srcpkgs/touchegg/patches/musl.patch [hunk 2]
```diff
@@ -0,0 +1,24 @@
+From da9a3a95cfe9965ff6e15e3a0e1a98f8840a34d7 Mon Sep 17 00:00:00 2001
```
Comment (ericonr) [inline srcpkgs/touchegg/patches/musl.patch:1]: Very upstreamable, you should look into that :)

**Comment by ericonr** [comment]
`/etc/xdg/autostart/touchegg.desktop` this file is new and I think it will make all DEs automatically start `touchegg`, right? Not sure we should ship it...

**Comment by crater2150** [comment]
oh, right. I'm not using a DE, just a window manager, so I didn't notice. I removed it.

**Comment by ericonr** [comment]
I'm not sure if it's a musl or wayland thing (probably wayland, seems to happen with glibc as well), but the client segfaults with almost any movements.

**Comment by ericonr** [comment]
Also, please apply

```diff
--- a/srcpkgs/touchegg/template
+++ b/srcpkgs/touchegg/template
@@ -4,7 +4,7 @@ version=2.0.9
 revision=1
 build_style=cmake
 hostmakedepends="pkg-config"
-makedepends="libudev-devel libinput-devel pugixml-devel cairo-devel libX11-devel
+makedepends="eudev-libudev-devel libinput-devel pugixml-devel cairo-devel libX11-devel
  libXtst-devel libXrandr-devel libXi-devel glib-devel gtk+3-devel"
 short_desc="Multitouch gesture recognizer"
 maintainer="Enno Boland <gottox@voidlinux.org>"
@@ -15,6 +15,8 @@ checksum=efff0f2011078cea15f2f130d1501714f8e3f82b7e883e81eccf956ac7966d0e
 system_accounts="_touchegg"

 post_install() {
-       rm -rf ${DESTDIR}/etc/xdg
+       # we don't want the package being installed to mean it should be autostarted
+       rm -r ${DESTDIR}/etc/xdg/autostart
+
        vsv touchegg
 }
```

**Comment by ericonr** [comment]
Ah, it isn't expected to work: https://github.com/JoseExposito/touchegg/issues/455

**Pull request closed without merging**